app/assets/javascripts/OutdoorMap/map_build.js

```diff
@@ -4,7 +4,7 @@
 // Set the leaflet map with center and zoom-level
```

Comment:
I noticed some unused imports on top of this file, maybe you could remove them, even though you did not introduce them.